### DISCUSSION TITLE
Heighmaps file

### DISCUSSION THREAD

**Discussion by RagBom7** [xdjxsonnn/Fortnite-Heightmaps#5] [General]
Hey, I have a question: How many seasons height maps are you trying to get? I know that it says that youre working in the X season, but do you have planned working in the other chapters seasons?

**Comment by jasn7135** [comment]
I can think about how or what I'll do in the future, but it could be true that I do chapter 2

**Reply by RagBom7** [reply]
Ok! Thanks! It would help me a lot. 
Is there a way to help in the project?

**Reply by jasn7135** [reply]
What kind of season do you particularly want in the future?

**Reply by RagBom7** [reply]
Its sound pretty funny but if you can, all.
If you cant at least I would like if you upload the seasons in order, I mean if you finish the project with the S7C2 for example, it would must be the S1-6C2, you should probably think the same, so I dont need a concrete season. 
But I am not in rush, so no problem.

**Reply by jasn7135** [reply]
If you really want it, I can do it for you and for my community

**Reply by RagBom7** [reply]
I would really apreciate it if you do it, and I didnt found a other place with other exactly heighmaps like this ones.
So if you can do this, you would have the biggest heighmaps copilation.

**Reply by jasn7135** [reply]
But I just have the following problem: my tool with which I export most heightmaps doesn't work in versions above 9.10

**Reply by RagBom7** [reply]
Oh, is there something I can do?

**Reply by jasn7135** [reply]
Yes, you could write me your discord name

**Reply by RagBom7** [reply]
Ok, its RagBom7#6763

**Reply by jasn7135** [reply]
I sent you a request
